lib/useOnyx.ts

```diff
@@ -57,8 +57,39 @@ type ResultMetadata = {
 
 type UseOnyxResult<TKey extends OnyxKey, TValue> = [CachedValue<TKey, TValue>, ResultMetadata];
 
-function getCachedValue<TKey extends OnyxKey, TValue>(key: TKey, selector?: Selector<TKey, unknown, unknown>): CachedValue<TKey, TValue> | undefined {
-    return (OnyxUtils.tryGetCachedValue(key, {selector}) ?? undefined) as CachedValue<TKey, TValue> | undefined;
+type UseOnyxSelector<TKey extends OnyxKey, SelectorValue> = (data?: OnyxValue<TKey>) => SelectorValue;
+
+function getCachedValue<TKey extends OnyxKey>(key: TKey): OnyxValue<TKey> | undefined {
```

Comment:
Could you add a description for this function? Also mentioning why is different from `tryGetCachedValue` implementation.

Reply:
Done ✅